F1WM.UnitTests/Controllers/TracksControllerTests.cs

```diff
@@ -66,5 +68,65 @@ public async Task ShouldReturn404IfTrackRecordsNotFound()
 			serviceMock.Verify(s => s.GetTrackRecords(trackId, trackVersion, null), Times.Once);
 			Assert.IsType<NotFoundResult>(result.Result);
 		}
+
+		[Fact]
+		public async Task ShouldReturnTracks()
```

Comment:
Please verify if data is correct, i.e. the tracks that are returned by the service are returned by the controller.